### ISSUE TITLE
Torch + CUDA installation

### ISSUE THREAD

**Issue by kingjin94** [pytorch/pytorch#118373]
### 🐛 Describe the bug

If CUDA, torch, graphic card version, etc. don't match up it can the case `torch.cuda.is_available()` but this not means CUDA works as expected. I found I also needed to test some simple CUDA math such as:

```
t = torch.zeros((1000,)).to('cuda')
t2 = t*t
```

to get additional verification and make the install work.

I would suggest adding such a minimal test either to the getting started guide or maybe even inside the cuda.is_available().

### Versions

Various invalid combinations of torch + cuda + python + OS versions.

**Comment by ptrblck** [comment]
> or maybe even inside the cuda.is_available().

I don't think this should land in `torch.cuda.is_available()` as it will initialize a CUDA context, which we generally don't want in util. functions. 
Adding a minimal smoke test to the docs sounds fine.
